kernel/data-pipeline/core/src/main/java/org/apache/shardingsphere/data/pipeline/core/importer/DefaultImporter.java

```diff
@@ -79,11 +80,11 @@ public final class DefaultImporter extends AbstractLifecycleExecutor implements
     
     private volatile Statement batchDeleteStatement;
     
-    public DefaultImporter(final ImporterConfiguration importerConfig, final PipelineDataSourceManager dataSourceManager, final PipelineChannel channel,
+    public DefaultImporter(final ImporterConfiguration importerConfig, final ImporterConnector importerConnector, final PipelineChannel channel,
                            final PipelineJobProgressListener jobProgressListener) {
```

Comment:
`ImporterConnector` is imported here. Some questions:
1. Is there new type of Importer impl for CDC?
2. If yes, it's better to update related factory `DefaultImporterCreator`. If no, is it enough to reuse `DefaultImporter`?